requiam/commons.py

```diff
@@ -34,6 +34,39 @@ def figshare_stem(stem='', production=True):
     return stem_query
 
 
+def figshare_group(group, root_stem, production=True):
+    """
+    Purpose:
+      Construct Grouper figshare groups
+
+    :param group: str or int of group name. Cannot be empty
+    :param root_stem: str of associated stem folder for [group]
+    :param production: Bool to use production stem. Otherwise a stage/test is used. Default: True
+
+    :return grouper_group: str containing full Grouper path
+
+    Usage:
+      For active group, call as: figshare_group('active', '')
+        > 'arizona.edu:dept:LBRY:figshare:active'
+
+      For a quota group, call as: figshare_group('2147483648', 'quota')
+        > 'arizona.edu:dept:LBRY:figshare:quota:2147483648'
+      Note: group can be specified as an integer for quota cases
+
+      For a portal group, call as: figshare_group('sci_math', 'portal')
+        > 'arizona.edu:dept:LBRY:figshare:portal:sci_math'
+    """
+
+    if not group:
+        raise ValueError("WARNING: Empty [group]")
```

Comment:
Interesting thought. First thing, `group`'s type is a `str` and not a `NoneType`. In the suggestion, this would mean that it's an optional argument, but it is required. This handling is intended if one passes `group=''`

Reply:
Understood. Then I suggest defining the function as such so the default case can be an empty string. This would prevent the program from crashing without the custom warning if no parameter is passed (would that happen though?)

`(group= '', .... etc)`

Reply:
`figshare_group` is intended to retrieve a Grouper group name associated with the Grouper stem. Providing an empty string would just give the stem, which would be confusing. We have a separate function for `figshare_stem`

Reply:
Sounds good. I'll finish the feature and merge it into develop then.